src/components/Callout/Callout.scss

```diff
@@ -366,6 +366,7 @@
 // Disabled default box shadow.
 .ms-Callout-main {
   box-shadow: none;
+  @include focus-border();
```

Comment:
hmm this seems super weird. why would we want to have a focus border on the callout itself?